ClosedXML/Utils/OpenXmlHelper.cs

```diff
@@ -11,7 +11,7 @@ public static BooleanValue GetBooleanValue(bool value, bool defaultValue)
 
         public static bool GetBooleanValueAsBool(BooleanValue value, bool defaultValue)
         {
-            return value == null ? defaultValue : value.Value;
+            return value.HasValue ? defaultValue : value.Value;
```

Comment:
Since the ```value``` is not a ```Nullable<bool>``` but is a ```BooleanValue``` instance, this will cause ```NullReferenceException``` if ```value == null```.

Reply:
I think it should just be
```c#
 return value.HasValue ?  value.Value : defaultValue;
```

@Mafii, Can you please fix?

Reply:
@igitur I will this weekend. Thanks for the heads up and sorry for the inconvenience!

Reply:
The problem is quite simple: 

```
return value.HasValue ? defaultValue : value.Value;
```

This piece of code will return the default value instead of the value when the value is available, and the other way around. Flipping the two values after the ? will do it.